.github/workflows/storybook.yml

```
@@ -0,0 +1,64 @@
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4 v4.2.2
+        with:
+          persist-credentials: false
+
+      - name: Setup Node
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)